Dockerfile-base

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu
+ADD deploy/ ./deploy
+ADD settings/ ./settings
+
+
+ENV ENSEMBL_API_VERSION ${ENSEMBL_API_VERSION:-92} 
+
+# Set TEST_ENV to nothing to stop deploying a MariaDB along with your server
+ENV TEST_ENV true
```

Comment:
I don't see any volumes defined. Where is the mysql database living? Will it be destroyed across restarts?

Reply:
I'll discuss in known issues